node/how_to_with_npm/prisma.md

```diff
@@ -0,0 +1,253 @@
+# How to create a RESTful API with Prisma and Oak
+
+[Prisma](https://prisma.io) has been one of our top requested modules to work
+with in Deno. The demand is understandable, given that Prisma's developer
+experience is top notch and plays well with so many persistent data storage
+technologies.
+
+We're excited to show you how to use Prisma with Deno.
+
+In this How To guide, we'll setup a simple RESTful API in Deno using Oak and
+Prisma.
+
+Let's get started.
+
+1. Setup the application
+
+```shell
+mkdir rest-api-with-prisma-oak
+cd rest-api-with-prisma-oak
+deno run -A npm:prisma init
+```
+
+Update the `prisma/schema.prisma` with the following:
+
+```ts
+generator client {
+  provider = "prisma-client-js"
+  previewFeatures = ["deno"]
+  output = "../generated/client"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Dinosaur {
+  id          Int     @id @default(autoincrement())
+  name        String  @unique
+  description String
+}
+```
+
+Create a `.env` file and update the connection string with your actual
+PostgreSQL connection string.
+
+2. Create the database schema
+
+```shell
+deno run -A npm:prisma db push
```

Comment:
```suggestion
deno run -A npm:prisma@4.6 db push
```

It would be nice if prisma created a deno task like `deno task prisma` that had `deno run -A npm:prisma@4.6` in it, but alas.